### PR TITLE
mediaservices housekeeping - fixing tags, removing redundancies

### PR DESCRIPTION
**Pull request by zikalino**
Tag **package-2018-07** stays default.
Included files from 2018-07-01 in **package-2019-05-preview** as these files in 2019-05-01-preview were just exact copies referring to 2018-07-01 API. Removed redundant files.
Included streamingservice.json from 2019-05-01-preview into 2019-09-01-preview to make sure it's not lost when 2019-09-01-preview is released

### CODE REVIEW

File: specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-05-01-preview/Common.json [hunk 1]
```diff
@@ -1,80 +0,0 @@
-{
```
Comment (BrianBlum) [inline specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-05-01-preview/Common.json]: This file cannot be deleted from 2019-05-01-preview. It's still being referenced in the 2019-05-preview tag. And it needs to be in the same folder as swaggers that reference it.

**Review by BrianBlum — COMMENTED** [review]
Please hold the merge until another member of the Media Services team can look at this. I'll find a volunteer today.

**Comment by anuchandy** [comment]
requesting re-assign as this is ARM spec.

**Comment by JackTn** [comment]
Hi, @zikalino. The PR has be closed for a long time and it's related branch still exist. If no further used for over 14 days, I will delete this related branch. Please tell me if you still need this branch.

**Comment by JackTn** [comment]
Hi, @zikalino. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Pull request closed without merging**